pkg/relayerproxy/proxy/proxy.go

```diff
@@ -0,0 +1,111 @@
+package proxy
+
+import (
+	"context"
+
+	sdkclient "github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	accounttypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	blocktypes "pocket/pkg/client"
```

Comment:
It is actually because the package does not exist yet. This will definitely break the build; we could TODO-comment it or have a temporary interface for it, until it gets merged. WDYT

Reply:
Resolved now by commenting the BlockClient import and occurrences in the code.